### DISCUSSION TITLE
alist主页如何去掉盘符前的复选框

### DISCUSSION THREAD

**Discussion by zhsxh** [AlistGo/alist#6121] [Q&A]
![微信图片_20240301105036](https://github.com/alist-org/alist/assets/161790378/92308911-c5c4-4aa7-a605-c50daf832474)
如图所示，alist主页上，每个盘符前都有复选框，这导致无法通过双击进入网盘内，每次点击都成了选择该盘符。

**Comment by zhsxh** [comment]
求教如何去掉盘符前的复选框

**Reply by yyyytawa** [reply]
![image](https://github.com/alist-org/alist/assets/116234899/35511271-fdfa-40b9-ba6a-93fb18314371)
右下角 开关复选框

**Reply by zhsxh** [reply]
非常感谢
